dfastmi/gui/dialog_view_model.py

```diff
@@ -114,7 +125,8 @@ def save_plots(self):
     @save_plots.setter
     def save_plots(self, value):
         self._save_plots = value
-        self.model.section["SavePlots"] = str(value)
+        if self.model and self.model.section :
+            self.model.section["SavePlots"] = str(value)
```

Comment:
I don't understand why we have a local field ánd the same data on the model for each property.

Reply:
I agree, this needs to be resolved